ionoscloud/resource_dataplatform_cluster.go

```diff
@@ -27,16 +27,17 @@ func resourceDataplatformCluster() *schema.Resource {
 			"datacenter_id": {
 				Type:        schema.TypeString,
 				Description: "The UUID of the virtual data center (VDC) in which the cluster is provisioned",
-				ValidateDiagFunc: validation.ToDiagFunc(validation.All(validation.StringLenBetween(32, 63),
-					validation.StringMatch(regexp.MustCompile("^[0-9a-f]{8}-?[0-9a-f]{4}-?[0-9a-f]{4}-?[0-9a-f]{4}-?[0-9a-f]{12}$"), ""))),
+				ValidateDiagFunc: validation.AllDiag(validation.ToDiagFunc(validation.StringLenBetween(32, 63)),
```

Comment:
can `validation.IsUUID` be used here instead ?

Reply:
done